#!/usr/bin/python3

def raise_exception_msg(message=""):
    """ The function should raise a NameError exception with a message."""
    raise NameError(message)
